### ISSUE TITLE
Passing fastcgi_param or similar parameter to wp-cli

### ISSUE THREAD

**Issue by kmcintyre** [wp-cli/wp-cli#4768]
I have 26 separate wordpress databases running off the same wordpress install.

My wp-config.php uses a fastcgi_param passed in via nginx

include_once("/opt/wp_config/" . $_SERVER['SITE_KEY'] . ".php");

--nginx stanza--
location /#site# {
    alias /opt/wordpress;
    index index.php;
    try_files $uri $uri/ /#site#/#site#/index.php?$args;
    location ~ /#site#/.+.php$ {
        include fastcgi_params;
        fastcgi_split_path_info ^/#site#(.+\.php)(.*)$;
        fastcgi_index index.php;
        fastcgi_param SITE_KEY "#site_key#";
        fastcgi_param  SCRIPT_FILENAME    $document_root$fastcgi_script_name;
        fastcgi_pass 127.0.0.1:9000;
    }
}

Is there anyway to mimic the SITE_KEY or pass another value to wp that will allow me to keep my current setup while executing wp-cli?

Thanks in advance.

**Comment by schlessera** [comment]
GitHub issues are meant for enhancement requests and specific, reproducible bugs, not for general support questions. Bug reports need to be reproducible with a fresh WordPress install. If your issue only reproduces in your environment, then it's likely specific to your environment. For support options, please review http://wp-cli.org/#support

TL;DR: Just come to the Slack #cli team and we can discuss this.
